pkg/message/types.go

```diff
@@ -41,3 +41,10 @@ type SubscribeWrapper struct {
 	MessageType string    `json:"messageType,omitempty"`
 	Content     []byte    `json:"content,omitempty"`
 }
+
+type ParseResult struct {
```

Comment:
This type doesn't need to be exported. Its only usage is internal. Can potentially be defined in `/internal/annotators/http/handler/base.go`